src/plugins/dashboard/public/application/embeddable/grid/dashboard_grid.tsx

```diff
@@ -19,13 +19,24 @@ import React from 'react';
 import { Subscription } from 'rxjs';
 import ReactGridLayout, { Layout, ReactGridLayoutProps } from 'react-grid-layout';
 import { GridData } from '../../../../common';
-import { ViewMode } from '../../../services/embeddable';
+import { ViewMode, EmbeddableRenderedEvent } from '../../../services/embeddable';
 import { DASHBOARD_GRID_COLUMN_COUNT, DASHBOARD_GRID_HEIGHT } from '../dashboard_constants';
 import { DashboardPanelState } from '../types';
 import { withKibana } from '../../../services/kibana_react';
 import { DashboardContainer, DashboardReactContextValue } from '../dashboard_container';
 import { DashboardGridItem } from './dashboard_grid_item';
 
+export type DashboardDataEventStatus = 'done' | 'error';
```

Comment:
Would it make more sense to pull all these types related to the event out into their own file, so like, the dashboard container isn't importing from Dashboard Grid to just get those types? Maybe a good pattern for future events to follow?